src/dataVersion.ts

```diff
@@ -0,0 +1,43 @@
+/*
+This file is responsible for switching views in original database like `ojp`, `routeing`
+etc. from for example `ojp_2019_02_02` to `ojp_2019_02_03`
+So in case of data failure we can simply switch views between old still working databases
+ */
+import {Container} from "./cli/Container";
+import {viewsSqlFactory} from "../config";
+import {ojpViews} from "../config/gtfs/views";
+import {OfflineDataProcessor} from "./database/OfflineDataProcessor";
+
+const container = new Container();
+
+const f = async () => {
```

Comment:
wow, so expressive function name :D 
Is there a need for wrapping this code into a function ? 
Also, I can see this function use "container" object from the outer scope, if you want to have this function, I would suggest to inject container as an argument instead of getting it from outer scope.

Reply:
I can also move Container to the function scope. 
+ I wrap it into function because when we execute query on database which is async function we need to await for result.